ballerina/build.gradle

```diff
@@ -16,10 +16,28 @@
  * under the License.
  */
 
+import org.apache.tools.ant.taskdefs.condition.Os
+
+buildscript {
+    repositories {
+        maven {
+            url = 'https://maven.pkg.github.com/ballerina-platform/plugin-gradle'
+            credentials {
+                username System.getenv("packageUser")
+                password System.getenv("packagePAT")
+            }
+        }
+    }
+    dependencies {
+        classpath "io.ballerina:plugin-gradle:${project.ballerinaGradlePluginVersion}"
+    }
+}
```

Comment:
As a best practice, let's do all the plugin management inside the settings.gradle file. Check the example repo (Gmail).

Reply:
Fixed with [2f2d71b9726944ad42af3ecdae198f45b1687811](https://github.com/ballerina-platform/module-ballerinax-azure-cosmosdb/pull/112/commits/2f2d71b9726944ad42af3ecdae198f45b1687811)